docs/gambit/mutation-verifier.md

```diff
@@ -1,17 +1,16 @@
 # Using Gambit with the Prover
 
-This is the mutation verifier which
- checks that variants of the original
- solidity program do not pass the specification.
+The mutation verifier checks that variants of the original
+  Solidity program do not pass verification against a given specification.
 It uses mutations from the {doc}`Gambit <gambit>`
   mutation generator.
 It also allows users to include manually generated mutants.
 If a mutated program passes the specification,
-it may indicate that the specification is vacuous or not rigorous enough.
+  it may indicate that the specification is vacuous or not rigorous enough.
 In the rest of the document,
   we refer to both the mutation generator and the verifier as Gambit.
```

Comment:
do we have to refer to the two tools as gambit? Can't we separate if needed?

Reply:
Actually we call it certoraMuate or mutation verification, not Gambit, so that line is now deleted